source/globus/globus_platform.rst

```diff
@@ -0,0 +1,70 @@
+.. raw:: html
+
+   <style> .bgrnd1 {background-color:#DCDCDC; border-radius:5px} </style>
+
+.. role:: bgrnd1
+
+
+.. _globus_platform:
+
+Globus file and data sharing platform
+=====================================
+
+The Globus platform enables developers to provide robust file transfer, sharing and search capabilities within their own research data applications and services, while leveraging advanced identity management, single sign-on, and authorization capabilities.
+
+This document is a hands-on guide to the Globus file sharing platform. It should complement the official documentation at `docs.globus.org <https://docs.globus.org>`_.
+
+For all questions concerning the Globus file sharing platform, please contact the VSC Globus team at data@vscentrum.be.
+
+.. include:: globus_contact_information.rst
+
+We welcome your feedback, comments and suggestions for improving the Globus Tutorial.
+
+
+.. _access:
+
+Access
+------
+
+.. include:: access.rst
```

Comment:
Using actual includes makes this one monolithic page.  Would it makes sense to put each section in its own page?